### DISCUSSION TITLE
How to enable/disable damagelogs?

### DISCUSSION THREAD

**Discussion by DasMaffin** [BadgerCode/tttdamagelogs#118] [Q&A]
![image](https://github.com/user-attachments/assets/a75774f1-0a9a-4167-81c8-4655c113b565)
On this image you can see that there is no damage in the damagelogs. I dont know how I turned it off and would like to know how so I can turn it back on.

**Comment by BadgerCode** [comment]
Hello!

You might have damage filtered out from the displayed logs.

There should be an **"Edit filters"** button at the top.
![Preview](https://i.imgur.com/9eFdulZ.png)

That should allow you to change which items are shown in the displayed logs, such as damages.

**Reply by DasMaffin** [reply]
![image](https://github.com/user-attachments/assets/d137a627-6f1f-4e25-be2e-259e49c8b149)
Sadly its not as easy as that. I have everything enabled, it should show ~6 legshot hits with a deagle, but it shows nothing sadly

This doesnt just happen on this server either. Also happens on my testserver as well.
